pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -24,7 +24,7 @@ import (
 	"time"
 
 	authenticationv1 "k8s.io/api/authentication/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
I didn't even notice that changed at first, because it was done automatically by my editor. `goimports` does this. Looking through the repo, it is used like this all over, so I didn't go make and remove these changes.